shortify/handlers.go

```diff
@@ -44,7 +45,13 @@ func createRedirectHandler(response http.ResponseWriter, request *http.Request)
 			return
 		}
 
-		redir, err := FindOrCreateRedirect(params.Url)
+		url, err := url.Parse(params.Url)
+		if err != nil || url.Scheme == "" {
+			renderError(response, HTTPUnprocessableEntity, "Invalid url")
+			return
+		}
```

Comment:
would it make sense to extact this to `validateUrl`" or `validUrl?` so you'd get:

```go
if !validUrl(params.Url)  return

redir, err := Find...
```